### ISSUE TITLE
OddNews Issue 11

### ISSUE THREAD

**Issue by SondraE** [oddbird/oddleventy#568]
## OddNews 11 Preview Link
https://us19.admin.mailchimp.com/campaigns/edit?id=16941425

____
- [ ] @SondraE [Making Websites More Intuitive](https://www.oddbird.net/2023/11/09/unintuitive-objects/) - Write another, shorter article (or video with Stacy?) with another example of unintuitive objects in a website and how to fix them (link between posts) 
- [x] @SondraE Proposal for how to turn the newsletter to a UX design and front of the front-end focus
- [x] @stacyk Review
- [x] @mirisuzanne Review
- [x] @SondraE Post to oddsite
- [x] @SondraE Post link on socials (https://us19.campaign-archive.com/?u=80219aa68d7bad77b9fd2eb93&id=edcb11869c)

**Comment by SondraE** [comment]
**OddNews redesign and content proposal**
https://www.figma.com/file/X4ImIvSDk0BMCGG4maEBO4/New-Focus-Proposal?type=design&node-id=1-2&mode=design&t=tp6RAUVFvR5lW3tD-0

**Comment by mirisuzanne** [comment]
@SondraE this (and the redesign) looks good. Can we include a mastodon link in addition to X?

I'm also curious why we don't link article headings, only buttons?

At some point, I think it might be nice to include a short note from one of us in each oddnews - just something we're thinking about. Adds a a personal touch to the email itself.

**Comment by SondraE** [comment]
@mirisuzanne 
MailChimp's built in share module doesn't have many options and doesn't allow custom social links. I'd need to build that section custom to get other socials in there. I'll keep that in mind for the next issue. We do have social links in the footer that include a wider range of our socials. For now, I've removed the social shares near the top and limited that section to just a 'share with friend' email link.

I've linked article headings.

Yep. I think it's a good idea to go in a more personal direction with the newsletter. My plan for this issue of the newsletter had been to make it more personal and more focused around a single topic, but I'll save that for a month when I'm not in and out so much.
